tests/unit/network/probe/ProbeServer.js

```diff
@@ -0,0 +1,169 @@
+const assert = require('assert');
```

Comment:
Same for the file name, should be `HealthProbeServer.js` (or `HealthProbeServer.spec.js` to distinguish better test files from code files when using code indexers).